sdk/helper/testhelpers/schema/response_validation.go

```diff
@@ -126,3 +127,42 @@ func GetResponseSchema(t *testing.T, path *framework.Path, operation logical.Ope
 
 	return &schemaResponses[0]
 }
+
+func ResponseValidatingCallback(t *testing.T) func(logical.Backend, *logical.Request, *logical.Response) {
```

Comment:
Could you please add a short comment describing the method

Reply:
done